composer.json

```diff
@@ -23,7 +23,7 @@
     "ext-json": "*",
     "contributte/psr7-http-message": "~0.7.0",
     "doctrine/annotations": "^1.13",
-    "koriym/attributes": "^1.0",
+    "koriym/attributes": "1.x-dev#2b0349d",
     "nette/utils": "^3.0.3"
   },
```

Comment:
@koriym Btw, I'm testing https://github.com/koriym/Koriym.Attributes/pull/11 here and everything passes well :+1: 

I've removed the custom DualReader implementation from here and used the native one.